src/models/base.js

```diff
@@ -67,18 +72,20 @@ class base {
         path += `&page=${page}&per_page=${perPage}`;
 
         return new Promise((resolve, reject) => {
-            request(`${this.url}${path}`, {
-                json: true,
-                insecure: this._insecure,
-                proxy: this._proxy,
-                resolveWithFullResponse: true,
-                headers: {
-                    'PRIVATE-TOKEN': this.token
-                }
-            }).then(response => {
-                if (this.config.get('_createDump')) this.setDump(response, key);
-                resolve(response);
-            }).catch(e => reject(e));
+            throttle(() => {
+              request(`${this.url}${path}`, {
+                  json: true,
+                  insecure: this._insecure,
+                  proxy: this._proxy,
+                  resolveWithFullResponse: true,
+                  headers: {
+                      'PRIVATE-TOKEN': this.token
+                  }
+              }).then(response => {
+                  if (this.config.get('_createDump')) this.setDump(response, key);
+                  resolve(response);
+              }).catch(e => reject(e));
+            })
```

Comment:
same suggestion as above. Move throttle one line up and un-indent the request lines to minimize the diff.